### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henrymbuguakiarie**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by henrymbuguakiarie** [comment]
I created this PR  because the original [PR](https://github.com/Azure-Samples/ms-identity-ciam-native-auth-android-sample/pull/18) was accidentally closed.

File: README.md [hunk 1]
```diff
@@ -1,229 +1,124 @@
...
+### Step 9: Call a protected web API
 
-   If you don't have any devices configured, you need to either create an Android Virtual Device to use the Android Emulator or connect a physical device.
+Follow the steps in [Sign in users and call an API in a sample Android mobile app by using native authentication](https://learn.microsoft.com/entra/external-id/customers/sample-native-authentication-android-sample-app-call-web-api) to sign in users and call a protected API in the Android sample mobile app.
 
-1. Select **Run** button.
+## Key concepts
 
-1. The app will open on the email and one-time passcode screen.
+Open `app/src/main/res/raw/native_auth_sample_app_config.json` file and you find the following lines of code:
```
Comment (Yuki-YuXin) [inline README.md:71]: Move comments from another PR
image

**Review by Yuki-YuXin — APPROVED** [review]
I am all good with the readme file and thanks for your work :)

File: README.md [hunk 2]
```diff
@@ -1,229 +1,124 @@
...
+* [Project setup](#project-setup)
+* [Key concepts](#key-concepts)
+* [Reporting problems](#reporting-problems)
+* [Contributing](#contributing)
 
-This is a sample application used to demonstrate the CIAM extension of the MSAL framework that is currently in private preview. It incorporates private extensions of the [MSAL Android](https://github.com/AzureAD/msal-android-native-auth-preview) and [MSAL Android Common](https://github.com/AzureAD/msal-common-for-android-native-auth-preview) libraries.
+## Overview
 
-## Introduction
+This sample Android application demonstrates how to handle sign-up, sign-in, sign-out, and password reset scenarios using Microsoft Entra External ID for customers. You can configure the sample to call a protected web API.
```
[the remark this answers is not in the document]
Comment (henrymbuguakiarie) [reply README.md:13]: I've updated this.

File: README.md [hunk 3]
```diff
@@ -1,229 +1,124 @@
...
-1. Select **Add application**.
-1. Select the application from the list such as _ciam-client-app_ or use the search box to find the application, and then select it.
+* <a href="https://developer.android.com/studio/archive" target="_blank">Android Studio Dolphin | 2021.3.1 Patch 1</a>.
+* Microsoft Entra External ID for customers tenant. If you don't already have one, <a href="https://aka.ms/ciam-free-trial?wt.mc_id=ciamcustomertenantfreetrial_linkclick_content_cnl" target="_blank">sign up for a free trial</a>.
 
-1. Choose **Select**.
+## Project setup
 
-## Clone sample Android mobile application
+To enable your application to authenicate users with Microsoft Entra, Microsoft Entra ID for customers must be made aware of the application you create. The following steps show you how to:
```
[the remark this answers is not in the document]
Comment (henrymbuguakiarie) [reply README.md:31]: I've updated to reflect the changes.

File: README.md (hunk 1 above)
Comment (henrymbuguakiarie) [reply README.md:71]: I've updated to reflect the changes.
